pkg/ddc/alluxio/utils.go

```diff
@@ -186,28 +186,48 @@ func isPortInUsed(port int, usedPorts []int) bool {
 	return false
 }
 
-func (e *AlluxioEngine) parseRuntimeImage() (image, tag string) {
-	var (
-		defaultImage = "registry.cn-huhehaote.aliyuncs.com/alluxio/alluxio"
-		defaultTag   = "2.3.0-SNAPSHOT-2c41226"
-	)
+func (e *AlluxioEngine) parseRuntimeImage(image string, tag string, imagePullPolicy string) (string, string, string) {
+	if len(imagePullPolicy) == 0 {
+		imagePullPolicy = "IfNotPresent"
```

Comment:
Unified one configuration is better ? 
e.g. `common.DefImagePullPolicy`

Reply:
Fix it. Thanks.